ts/features/wallet/component/PagoPaPaymentCapability.tsx

```diff
@@ -83,28 +97,52 @@ const availabilityBadge = (badgeType: PaymentSupportStatus) => {
  *   - The card is not abilitated to pay on IO
  * @param props
  */
-function PagoPaPaymentCapability(props: Props): React.ReactElement {
+const PagoPaPaymentCapability: React.FC<Props> = props => {
+  const onOpenLearnMoreAboutInAppPayments = () =>
+    openWebUrl(IN_APP_PAYMENTS_LEARN_MORE_VIDEO_URL);
+
+  const { present } = useIOBottomSheet(
+    <View>
+      <Markdown>
+        {I18n.t("wallet.methods.card.pagoPaCapability.bottomSheetBody")}
+      </Markdown>
+      <ButtonDefaultOpacity
+        onPress={onOpenLearnMoreAboutInAppPayments}
+        style={styles.bottomSheetCTA}
+        onPressWithGestureHandler={true}
+      >
+        <Link>
+          {I18n.t("wallet.methods.card.pagoPaCapability.bottomSheetCTA")}
+        </Link>
+      </ButtonDefaultOpacity>
+    </View>,
+    I18n.t("wallet.methods.card.pagoPaCapability.bottomSheetTitle"),
+    300
+  );
+
   return (
-    <View style={styles.row}>
-      <View style={styles.left}>
-        <H4
-          weight={"SemiBold"}
-          color={"bluegreyDark"}
-          testID={"PagoPaPaymentCapabilityTitle"}
-        >
-          {I18n.t("wallet.methods.card.pagoPaCapability.title")}
-        </H4>
-        <H5
-          weight={"Regular"}
-          color={"bluegrey"}
-          testID={"PagoPaPaymentCapabilityDescription"}
-        >
-          {I18n.t("wallet.methods.card.pagoPaCapability.description")}
-        </H5>
+    <TouchableOpacity onPress={present}>
```

Comment:
imho in this case there's no need to use a component from 3rd party library, let's use `TouchableDefaultOpacity` from `ts/components/TouchableDefaultOpacity.tsx` which is based on `TouchableOpacity` from `react-native` package